Src/Plugins/Tests.AzureDevOpsConfiguration/TestData/BuildDefinitionSecurity/ExpectedOutputs/AZC101.builddefinition.DontMakeSecretsAvailableToForkedBuilds.sarif

```diff
@@ -5,31 +5,47 @@
     {
       "tool": {
         "driver": {
-          "name": "testhost",
-          "organization": "Microsoft Corporation",
-          "product": "Microsoft.TestHost",
-          "fullName": "testhost 15.0.0.0",
-          "version": "15.0.0.0",
-          "semanticVersion": "15.0.0",
-          "rules": [
-            {
-              "id": "AZC101/240",
-              "name": "DontMakeSecretsAvailableToForkedBuilds",
-              "fullDescription": {
-                "text": "Do not make secrets available to builds for fork of public repository."
-              },
-              "messageStrings": {
-                "NotApplicable_InvalidMetadata": {
-                  "text": "'{0}' was not evaluated for check '{1}' because the analysis is not relevant for the following reason: {2}."
+          "name": "Spmi",
```

Comment:
>"name": "Spmi", [](http://example.com/codeflow?start=10&length=15)

Nice to see this old testing bug cleared up! Where is the change that updated this? Asking to make sure this wasn't achieved through a local configuration change.